package.json

```diff
@@ -47,7 +47,7 @@
     "tap": "^10.0.1"
   },
   "engines": {
-    "node": ">=0.10",
+    "node": ">=8",
     "npm": ">=2"
```

Comment:
Btw, by enforcing node8 npm5 becomes the de facto min npm version – unless people explicitly downgrade.